test/api/controllers/collectionController.test.js

```diff
@@ -112,43 +112,44 @@ describe('Test: collection controller', () => {
 
     it('should return a dedicated error if the index does not exist', () => {
       kuzzle.indexCache.exists.resolves(false);
-
-      return should(collectionController.getSpecifications(request))
-        .rejectedWith(
-          PreconditionError,
-          {message: `The index '${index}' does not exist`});
+      
+      return collectionController.getSpecifications(request)
+        .should.be.rejectedWith(PreconditionError, {message: `The index ${index} does not exist.`});
```

Comment:
Here and for all other occurences: prefer `should(` over its `Object` override form.

Reply:
Okay
but why ?

Reply:
For consistency with the other tests